src/graph.rs

```diff
@@ -1352,7 +1423,10 @@ pub(crate) enum ModuleSlot {
   /// When trying to load or parse the module, an error occurred.
   Err(ModuleError),
   /// An internal state set when loading a module asynchronously.
-  Pending,
+  Pending {
+    /// If loading an asset, which won't load the content into memory.
```

Comment:
I'm not sure I understand this - how are we gonna do a bytes import without loading a file into memory?